resteasy-feature-pack/galleon-feature-pack/pom.xml

```diff
@@ -96,6 +96,12 @@
                                     </includes>
                                     <filtering>true</filtering>
                                 </resource>
+                                <resource>
+                                    <directory>${common.resources.directory}</directory>
+                                    <includes>
+                                        <exclude>configs/**/model.xml</exclude>
```

Comment:
@jamezp , we still have an exclude? (I suppose could be named anything).

Reply:
Oh man, I need to pay more attention in the mornings haha. Thank you @jfdenise